format_spec/enumeration.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Enumerations
+---
+
+## Main Structure
+
+```
+my_array                                    # array folder
+   |  ...
+   |_ schema                      # ArraySchema directory named `__schema`
+         |_ enumerations          # Enumeration directory named `__enumerations`
+         |      |_ enumeration    # enumeration data with names `__t1_t2_uuid_v`
+
+
+Enumeration data is stored in a subdirectory of the [array schema][./array_schema.md]
+directory. Enumerations are stored using [Generic Tiles][./generic_tile.md].
+Data stored in the generic tile follows the current format as of version 19.
+
+| **Field** | **Type** | **Description** |
+| :--- | :--- | :--- |
+| Version number | `uint32_t` | Format version number of the generic tile |
```

Comment:
This was implemented quite poorly before by attempting to re-use the format_version as the enumerations version. Since then I've split the naming and versioning so that things are more clear.

You were quite right that the format version is already part of the GenericTileIO header so if we really needed it, we could have gotten it there.

This however is a version for the Enumeration data itself. It now starts at 0, and if we ever have to change this (de)?serialization implementation for the *contents* of the GenericTileIO "payload" we can do that by using this Enumerations specific version.